modules/p2/ding_serendipity_ting_entity/ding_serendipity_ting_entity.module

```diff
@@ -103,20 +107,20 @@ function ding_serendipity_ting_entity_ting_object_author_serendipity_add($contex
     return $results;
   }
 
-  foreach ($creators as & $creator) {
-    $creator = 'facet.creator="' . $creator . '"';
+  // Find related content from any of the authors.
+  $field_filters = [];
+  foreach ($creators as $creator) {
+    $field_filters[] = new TingSearchFieldFilter(TingSearchCommonFields::AUTHOR, $creator, '=', BooleanStatementInterface::OP_OR);
```

Comment:
The or statement is on the filter? How does that work with mixed AND and ORs?

Wouldn't it be smarter with AND and OR objects that contain field filters?

And why aren't we duplicating the $query->condition(...) pattern of EntityFieldQuery?

Reply:
I'll do that in a follow up ticket!